release-notes/release-notes5014.md

```diff
@@ -54,3 +54,25 @@ For a list of bug fixes included in this release, see: [Changelog - Version 5.0.
 When using an editor larger than the size of the screen, the find and replace dialog may disappear off the top
 of the screen and users will need to scroll back up to find the next match in the content. A fix will be provided in an upcoming
 release of TinyMCE.
+
+## Errata&#58; TinyMCE 5.0.14 Premium Plugins
+
+This errata documents updates to the TinyMCE 5.0.14 Premium Plugins, and was released on the 12th September, 2019. Future releases of TinyMCE will also include these changes.
+
+### Errata&#58; updates and enhancements
+
+#### PowerPaste
+
+PowerPaste 5.1.0 includes a significant update to how tables are handled when pasted from Microsoft Word or Excel. The new version produces cleaner table HTML which better conforms with HTML5 standards. 
```

Comment:
Maybe make `PowerPaste` bold, or remove the bold markup around the product names in the Media Embed section?